test/functional/cmdLineTests/J9security/playlist.xml

```diff
@@ -50,6 +50,12 @@
 
 	<test>
 		<testCaseName>cmdLineTester_J9securityTests</testCaseName>
+		<disables>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/20563</comment>
+				<version>24+</version>
```

Comment:
@llxia is this the preferred way to disable tests from running in future versions? The security tests should run for JDK 11-23 only.

Reply:
`disable ` is for temporary excludes. In this case, we should set `<version>[8, 23]</version>` . Example code: https://github.com/adoptium/TKG/blob/master/examples/jdkVersion/playlist.xml#L39